release-calendar/src/development/developmentData.ts

```diff
@@ -0,0 +1,45 @@
+import {Connection} from 'typeorm';
```

Comment:
All files need the copyright / license blurb up top. In `ampproject/amphtml`, it gets added automatically but we haven't gotten around to automating it here yet.

Reply:
I'll just manually add it then :)